market_simulation/user_json_renderer.py

```diff
@@ -0,0 +1,17 @@
+import json
+from rest_framework.renderers import JSONRenderer
+
+
+class UserJSONRenderer(JSONRenderer):
+    charset = "utf-8"
+
+    def render(self, data, media_type=None, renderer_context=None):
+        errors = data.get("errors", None)
```

Comment:
```suggestion
        errors = data.get("errors")
```
If errors is not present, the default value by .get() will be None.